components/Button.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+interface ButtonProps {
+  label?: string;
+  children?: React.ReactNode;
+  onClick?: () => void;
+  disabled?: boolean;
+  color?: string;
+  backgroundColor?: string;
+  borderRadius?: string;
+  href?: string;
+  className?: string;
+  hoverColor?: string;
+  padding?: [string, string];
+  margin?: [string, string];
+}
+
+const Button: React.FC<ButtonProps> = ({
+  label,
+  children,
+  onClick,
+  disabled = false,
+  color = 'text-white',
```

Comment:
can we get rid of the `color`, `backgroundColor`, `borderRadius`, `padding` and `margin` classes, the reasoning I think this is beneficial is the less props we have the more control we have and they are all equivlent to `className`.

If we are keeping the others then I think we should use that tailwind library you were talking about.

Reply:
fixed